### PR TITLE
Update tutorial.rst

### PR DESCRIPTION
**Pull request by petersmythe**
Spelling & grammar fixes

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/main/CONTRIBUTING.md).
- [x] I have sent a [Contribution Licence Agreement](https://docs.geoserver.org/latest/en/developer/policies/committing.html) (not required for small changes, e.g., fixing typos in documentation).
- [x] First PR targets the `main` branch (backports managed later; ignore for branch specific issues).
- [ ] All the build checks are green ([see automated QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html)).

For core and extension modules:

- [ ] New unit tests have been added covering the changes.
- [ ] [Documentation](https://github.com/geoserver/geoserver/tree/main/doc/en/user/source) has been updated (if change is visible to end users).
- [ ] The [REST API docs](https://github.com/geoserver/geoserver/tree/main/doc/en/api/1.0.0) have been updated (when changing configuration objects or the REST controllers).
- [ ] There is an issue in the [GeoServer Jira](https://osgeo-org.atlassian.net/browse/GEOS/summary) (except for changes that do not affect administrators or end users in any way).
- [ ] Commit message(s) must be in the form ``[GEOS-XYZWV] Title of the Jira ticket``.
- [ ] Bug fixes and small new features are presented as a single commit.
- [ ] Each commit has a single objective (if there are multiple commits, each has a separate JIRA ticket describing its goal).

### CODE REVIEW

File: doc/en/user/source/extensions/vectortiles/tutorial.rst [hunk 1]
```diff
@@ -27,7 +27,7 @@ Vector tile formats
 
 GeoServer can also produce vector tiles in three formats: GeoJSON, TopoJSON, and MapBox Vector (MVT). These are also supported by OpenLayers and other clients.
 
-.. warning:: When using vector tiles, be sure to use an up-to-date client. Older clients do not support all vector tiles capabilites and may result in rendering errors. We recommend using the latest version of OpenLayers (Currently v5.3.0).
+.. warning:: When using vector tiles, be sure to use an up-to-date client. Older clients do not support all vector tiles capabilities and may result in rendering errors. We recommend using the latest version of OpenLayers (Currently v5.3.0).
```
Comment (bradh) [inline doc/en/user/source/extensions/vectortiles/tutorial.rst:30]: Could update version here, perhaps be a bit more general.

Comment (jodygarnett) [reply doc/en/user/source/extensions/vectortiles/tutorial.rst:30]: We should probably just remove this warning as I am sure active open layers branches are fine

Comment (jodygarnett) [inline doc/en/user/source/extensions/vectortiles/tutorial.rst:30]:
```suggestion
```

Comment (mprins) [reply doc/en/user/source/extensions/vectortiles/tutorial.rst]: 👍🏽

**Comment by mprins** [comment]
docs builds passed, merging

**Pull request merged**